gto-support-gson/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="org.ccci.gto.android.common.gson"/>
```

Comment:
I could have made this a jar currently, which wouldn't have had that constraint. But the shared gradle build script is currently setup to build everything as android libraries.